backends/cadence/aot/compiler.py

```diff
@@ -16,18 +18,56 @@
     ReplaceScalarTensorWithFullPass,
     ReplaceSqueezeAndUnsqueezeWithViewPass,
 )
+from executorch.backends.cadence.aot.quantizer.fusion_pass import QuantFusion
+from executorch.backends.cadence.aot.quantizer.quantizer import CadenceQuantizer
 from executorch.backends.cadence.aot.utils import model_is_quantized
 from executorch.exir import EdgeCompileConfig, EdgeProgramManager, to_edge
+from torch._export import capture_pre_autograd_graph
 from torch.ao.quantization.pt2e.export_utils import model_is_exported
+from torch.ao.quantization.quantize_pt2e import convert_pt2e, prepare_pt2e
 
 from torch.export import export
 from torch.export.exported_program import ExportedProgram
 
+# pyre-ignore[33]: `InputsArgsType` cannot alias to `Any`.
+InputsArgsType = Tuple[Any, ...]
+
+
+def quantize_pt2(
+    model: torch.nn.Module,
+    inputs: InputsArgsType,
+) -> torch.fx.GraphModule:
+    """
+    Instantiate the CadenceQuantizer (PTQ), prepare, convert and fuse the model.
+    Returns a GraphModule with the quantized model.
+    """
+    # Quantizer
+    quantizer = CadenceQuantizer()
+
+    # Export with dynamo
+    model_exp = capture_pre_autograd_graph(model, inputs)
+
+    # Prepare
+    prepared_model = prepare_pt2e(model_exp, quantizer)
+
+    # Calibrate
+    prepared_model(*inputs)
```

Comment:
You'll need to calibrate with more than one sample normally for good PTQ accuracy. Is this API meant more for testing?

Reply:
yup this is all dummy for now. We don't even look at quantization accuracy, only implementation accuracy (ref outputs are from the converted forward model).